oelint_adv/rule_base/rule_var_src_uri_checksum.py

```diff
@@ -0,0 +1,73 @@
+from oelint_parser.cls_item import Variable
+from oelint_adv.cls_rule import Rule
+from oelint_parser.helper_files import get_scr_components
+from oelint_parser.parser import INLINE_BLOCK
+
+
+class VarSRCUriOptions(Rule):
```

Comment:
That should be named differently as there is already a class by that name in the current ruleset

Reply:
I changed my code like this. Thanks.